php-frontend/src/test/java/org/sonar/php/parser/expression/FunctionCallParameterListTest.java

```diff
@@ -34,7 +34,11 @@ public void test() {
       .matches("(& $p)")
       .matches("(...$p)")
       .matches("(yield $a)")
-      .matches("($p1, & $p2, ...$p3, yield $p4)");
+      .matches("($p1, & $p2, ...$p3, yield $p4)")
+      .notMatches("('title', 'body','comments',)")
```

Comment:
According to this [rfc](https://wiki.php.net/rfc/trailing-comma-function-calls) this code should "matches" and not the opposite!!!

Reply:
indeed, sorry!